101-hdinsight-hbase-replication-geo/azuredeploy.json

```diff
@@ -215,10 +215,10 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
-      "name": "[variables('vpnConnections').name1]",
       "type": "Microsoft.Network/connections",
-      "location": "[variables('clusterNode1').location]",
+      "name": "[variables('vpnConnections').name1]",
+      "apiVersion": "2015-06-15",
```

Comment:
update apiVersion

Reply:
updated

Reply:
Microsoft.HDInsight/clusters hasn't updated the api version for a long time.